### PR TITLE
chore: policy page

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

### CODE REVIEW

**Comment by allella** [comment]
Thanks for creating these.

I would suggest removing the commented out HTML.  I plan to reword some things once it's merged in and we can reference the fCC pages if we need to see that content.

`/* commeted these because we don&apost have way to do them, yet.`

**Review by ojeytonwilliams — COMMENTED** [review]
The redirect works just fine. When we have Chapter specific text, this should be fine for the MVP.

File: client/src/pages/policy/[policyId].tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { PolicyPage } from 'modules/policy';
```
Comment (Sboonny) [inline client/src/pages/policy/[policyId].tsx:1]:
```suggestion
import { PolicyPage } from '../../modules/policy';

File: client/src/pages/policy/index.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { PolicyPage } from 'modules/policy';
```
Comment (Sboonny) [inline client/src/pages/policy/index.tsx:1]:
```suggestion
import { PolicyPage } from '../../modules/policy';

File: client/src/pages/policy/[policyId].tsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { PolicyPage } from '../../modules/policy';
+
+export default PolicyPage;
```
Comment (gikf) [inline client/src/pages/policy/[policyId].tsx:3]: This is regarding the file itself, not the code. Is there need for a dynamic route `/policy/{whatever}`?

**Review by gikf — APPROVED** [review]
Unless text needs to be changed in this PR, this looks good to me :+1:

**Comment by allella** [comment]
Go ahead and I'll take a look at the text once it's merged in and open a PR with any tweaks. Thanks

**Pull request merged**